fdbserver/workloads/MetaclusterManagementWorkload.actor.cpp

```diff
@@ -771,7 +799,14 @@ struct MetaclusterManagementWorkload : TestWorkload {
 				ASSERT(exists);
 				return Void();
 			} else if (e.code() == error_code_metacluster_no_capacity) {
-				ASSERT(!hasCapacity && !exists);
+				ASSERT(!exists);
```

Comment:
For `createTenant`, we should also check that either the cluster was chosen by name or that it supports auto-assignment after the tenant is successfully created

Reply:
Done. We actually need to check:
1. assignClusterAutomatically is false
2. does the tenant belong to a tenant group to which a cluster has already been assigned?